### DISCUSSION TITLE
Master nodes stop working after reboot

### DISCUSSION THREAD

**Discussion by GiovanniBarile** [zifeo/terraform-openstack-rke2#57] [Q&A]
Hello, i managed to configure a HA cluster (3 masters + 3 workers)
After rebooting one (or more) master, the same won't start anymore. 

This is the log : 

● rke2-server.service - Rancher Kubernetes Engine v2 (server)
     Loaded: loaded (/usr/local/lib/systemd/system/rke2-server.service; enabled; vendor preset: enabled)
     Active: activating (auto-restart) (Result: exit-code) since Wed 2024-11-13 17:09:44 UTC; 4s ago
       Docs: https://github.com/rancher/rke2#readme
    Process: 1625 ExecStartPre=/bin/sh -xc ! /usr/bin/systemctl is-enabled --quiet nm-cloud-setup.service (code=exited, status=0/SUCCESS)
    Process: 1627 ExecStartPre=/sbin/modprobe br_netfilter (code=exited, status=0/SUCCESS)
    Process: 1628 ExecStartPre=/sbin/modprobe overlay (code=exited, status=0/SUCCESS)
    Process: 1629 ExecStart=/usr/local/bin/rke2 server (code=exited, status=1/FAILURE)
    Process: 1639 ExecStopPost=/bin/sh -c systemd-cgls /system.slice/rke2-server.service | grep -Eo '[0-9]+ (containerd|kubelet)' | awk '{print $1}' | xargs -r kill (code=exited, status=0/SUCCESS)
   Main PID: 1629 (code=exited, status=1/FAILURE)
        CPU: 134ms

And when i try to run the rke2 server in debug mode i get this : 

sudo /usr/local/bin/rke2 server --debug
WARN[0000] not running in CIS mode
INFO[0000] Applying Pod Security Admission Configuration
INFO[0000] Starting rke2 v1.30.3+rke2r1 (e742dc53b463d205773cfb25633309671ce6777c)
FATA[0004] starting kubernetes: preparing server: failed to get CA certs: Get "https://192.168.42.4:9345/cacerts": dial tcp 192.168.42.4:9345: connect: no route to host

Thank you!

**Comment by zifeo** [comment]
@GiovanniBarile Thanks for the report. Can you `curl --insecure https://192.168.42.4:9345/cacerts` from that server and also the others and tell me whether it is successful or not (do not share the CA when successful)?
